compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIELinalgFunctionOutlining.cpp

```diff
@@ -88,6 +88,8 @@ void AMDAIELinalgFunctionOutliningPass::runOnOperation() {
   DenseMap<Operation *, std::string> computeOpToOutlinedFuncMap;
   SmallVector<Operation *> toBeErased;
   moduleOp.walk([&](linalg::LinalgOp computeOp) {
+    if (isa<linalg::CopyOp, linalg::FillOp>(computeOp))
```

Comment:
This is related to my suggestion in https://github.com/nod-ai/iree-amd-aie/pull/862#discussion_r1820847047 : suggestion there is to make sure we handle each op correctly with fail until we explicitly categorized it as 'must-outline' or 'must-not-outline'.

Could you have have 

auto mustOutline = [](Operation * op) { return isa<linalg::MatmulOp>(op); }
auto mustNotOutline = [](Operation * op) { return isa<linalg::CopyOp, linalg::FillOp>(op); }

And then 

```
if (mustNotOutline(computeOp)) return WalkResult::skip();
else if (!mustOutine(computeOp)) { // path to signal pass failure that the op is uncategorized; }
else { // continue as usual }
```

Above suggestion is not a robust way to do outlining, but at least we'll know exactly what is getting outlined and what isn't!